### DISCUSSION TITLE
很老的项目，如果是以Jsp为基座的应用呢？

### DISCUSSION THREAD

**Discussion by lvtraveler** [jd-opensource/micro-app#72] [Q&A]
很老的项目，如果是以Jsp为基座的应用呢？
micro-app有对应的配置方案吗，或者反过来，以react项目为基座，如何引入jsp项目作为子应用？

**Comment by bailicangdu** [comment]
可以，但需要注意的是micro-app只能在浏览器环境运行

**Reply by inier** [reply]
兼容性要求是基于项目自身的兼容性适配哈，microapp有什么特殊要求没
